test/e2e/crdvalidator_test.go

```diff
@@ -30,19 +27,20 @@ var _ = Describe("crd validation webhook", func() {
 			var crd *apiextensionsv1.CustomResourceDefinition
 
 			BeforeEach(func() {
-				crd = newTestingCRD(
-					util.GenName(defaultTestingCrdName),
-					[]apiextensionsv1.CustomResourceDefinitionVersion{{
-						Name:    "v1alpha1",
-						Served:  true,
-						Storage: true,
-						Schema: &apiextensionsv1.CustomResourceValidation{
-							OpenAPIV3Schema: &apiextensionsv1.JSONSchemaProps{
-								Type:        "object",
-								Description: "my crd schema",
+				crd = util.NewTestingCRD(defaultTestingCrdName, defaultTestingCrdGroup, true,
```

Comment:
does it make sense to move the `defaultTestingCrdName` and other associated constants to the util package as well?